src/lib/hash/streebog/streebog-precalc.cpp

```diff
@@ -0,0 +1,1680 @@
+/*
+ * Copyright (c) 2013, Alexey Degtyarev <alexey@renatasystems.org>.
+ * All rights reserved.
+ */
+
+#include <botan/streebog.h>
+
+namespace Botan {
+
+#if defined(BOTAN_TARGET_CPU_IS_LITTLE_ENDIAN)
```

Comment:
I had a feeling you would say this since I didn't see it done anywhere else in the code. Will update.